docs/internal/Element-Type-Specs.md

```diff
@@ -1,626 +1,3 @@
-_Ongoing authoritative reference of Template Question/Element types, including input parameters and response type (shape). Please ensure this document matches the current implementation at all times._
+_**Note: Element type reference is available in the [public docs](https://docs.mflow.nz/<put-the-exact-url-here-if-you-know-it>) **_
```

Comment:
`https://docs.mflow.nz/<put-the-exact-url-here-if-you-know-it>`

Do you know it? Well -- do ya?

Reply:
Oh! Haha, should be there now